Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
```

Comment:
### [bundler-audit](https://github.com/rubysec/bundler-audit#readme)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.9.0.1 → 0.9.1

[_~~change-log~~, [source-code](https://github.com/postmodern/bundler-audit), [gem-diff](https://my.diffend.io/gems/bundler-audit/0.9.0.1/0.9.1)_]





<details>
<summary>Commits</summary>

A change of **39** commits. See the full changes on [the compare page](https://github.com/rubysec/bundler-audit/compare/v0.9.0.1...v0.9.1).

These are the first 10 commits:
- (ebfca35) [Added a note about Ruby < 2.6 End-of-Life status.](https://github.com/rubysec/bundler-audit/commit/ebfca35b2e47eae7aaba42f0e37d61bb3e26049e)
- (af5a066) [Bump nokogiri from 1.11.6 to 1.12.5 in /spec/bundle/secure (…](https://github.com/rubysec/bundler-audit/commit/af5a066bd70cb01705350b3f477b176644066620)
- (f64883a) [Bump nokogiri from 1.11.6 to 1.12.5 in /spec/bundle/insecure…](https://github.com/rubysec/bundler-audit/commit/f64883a878d172722164495668ff205c21eb55c3)
- (198e1e8) [Add Ruby 3.1 to CI](https://github.com/rubysec/bundler-audit/commit/198e1e8db31d5975b8de578d4f06aaca1cdc0ba4)
- (781c4f3) [Fix assorted lints](https://github.com/rubysec/bundler-audit/commit/781c4f39261110c5c387e0d3b1f96a8e1260cd50)
- (245a2b6) [Switch the then clause to its own line](https://github.com/rubysec/bundler-audit/commit/245a2b6a8fbf1d40d6e18ef6d40b1e83a684b984)
- (5b22bd3) [Bump nokogiri from 1.12.5 to 1.13.3 in /spec/bundle/secure (…](https://github.com/rubysec/bundler-audit/commit/5b22bd3135978b93a1f3640e005c94afceb6c2c9)
- (9cb6320) [Use a less ambiguous separator for the version update sugges…](https://github.com/rubysec/bundler-audit/commit/9cb6320d06465eb641f580dbf19dd3ac75526368)
- (017d1f5) [Wrap suggested patched versions in quotes](https://github.com/rubysec/bundler-audit/commit/017d1f5eeb6aa8a3700e0bcfa8d57e5ea00a465e)
- (d2fe4c6) [Fixes --gemfile-lock example](https://github.com/rubysec/bundler-audit/commit/d2fe4c66881880d242faedf2ef9dcb9570354eb0)


</details>